Intel compiler CI keeps failing with CMake error

**Describe the bug**
The intel compiler CI keeps failing with a strange CMake error.

**To Reproduce**
Steps to reproduce the behavior:
1. Create a new PR
2. See error when Intel CI runs:
```
Run ctest --output-on-failure -C $BUILD_TYPE
  ctest --output-on-failure -C $BUILD_TYPE
  shell: /usr/bin/bash -e {0}
CMake Error: Unknown argument: -C
CMake Error: Run 'ctest --help' for all supported options.
Error: Process completed with exit code 1.
```

**Additional context**
`-- CMake version: 3.30.0`

--- a/.github/workflows/intel.yml
+++ b/.github/workflows/intel.yml
@@ -62,7 +62,7 @@ jobs:
       
     - name: Run tests
       working-directory: ${{runner.workspace}}/quokka/build
-      run: ctest --output-on-failure -C $BUILD_TYPE
+      run: ctest --output-on-failure
       
     - name: Upload test output
       if: always()

